src/core/diags/factory_diags.cpp

```diff
@@ -650,6 +653,34 @@ void Diags::TransmitDone(Error aError)
 
 #endif // OPENTHREAD_RADIO
 
+#if !OPENTHREAD_RADIO
+Error Diags::ProcessLinkRaw(uint8_t aArgsLength, char *aArgs[])
+{
+    Error error = kErrorNone;
+
+    if (aArgsLength == 0)
+    {
+        Output("raw link is %s\r\n", otPlatRadioIsEnabled(&GetInstance()) ? "enabled" : "disabled");
+    }
+    else if (strcmp(aArgs[0], "start") == 0)
+    {
+        error = otPlatRadioEnable(&GetInstance());
```

Comment:
Suggest using `Get<Radio>().Enable()` instead of direct `otPlatRadioEnable()` call?

Reply:
fixed.